src/commands/BootstrapCommand.js

```diff
@@ -28,16 +42,11 @@ export default class BootstrapCommand extends Command {
     this.progressBar.init(this.packages.length);
     this.logger.info("Linking all dependencies");
 
-    const ignore = this.flags.ignore || this.repository.bootstrapConfig.ignore;
-
-    // Get a filtered list of packages that will be bootstrapped.
-    const todoPackages = PackageUtilities.filterPackages(this.packages, ignore, true);
-
-    // Get a trimmed down graph that includes only those packages.
-    const filteredGraph = PackageUtilities.getPackageGraph(todoPackages);
```

Comment:
> unrelated to the PR

Generally speaking, smaller more targeted PRs are easier to review and therefore tend to get merged more quickly.

This PR adds much needed functionality, but it has gone through a number of iterations that each included a different set of unrelated changes.  These other changes may _also_ be valuable, but it would be easier to consider them on their own merit if they were showcased in dedicated PRs.